bebot/include/bebot/bebot.hpp

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <Eigen/Dense>
+
+namespace bebot {
+    class BeBOT {
+
+        private:
+            double _t0, _tf;
+            Eigen::MatrixXd _cpts; 
+
+        public:
+            BeBOT() = default;
+            ~BeBOT() = default;
+
+            BeBOT(const Eigen::Ref<Eigen::MatrixXd>& cpts);
+            BeBOT(const Eigen::Ref<Eigen::MatrixXd>& cpts, const double tf);
+            BeBOT(const Eigen::Ref<Eigen::MatrixXd>& cpts, const double t0, const double tf);
```

Comment:
I don't understand the purpose of multiple constructors. What happens if initial and final time are not specified? Scaled to 0..1? In that case, give the parameters default constructors.